src/models/application.js

```diff
@@ -0,0 +1,40 @@
+import { BaseModel } from './base.js';
+
+const SCHEMA = {
+    definitions: BaseModel.schema.definitions,
+    type: 'object',
+    properties: {
+        id: { type: 'string', maximum: 255 },
+        api_key: { type: 'string', maximum: 255 },
+        name: { type: 'string', maximum: 255 },
+        description: {
+            oneOf: [
+                { type: 'null' },
+                { type: 'string' },
+            ],
+        },
+        created: {
```

Comment:
Is `created` something filled by the server? If yes, we should add the type `null` too, otherwise new model instantiations will fail.